gapic/samplegen/samplegen.py

```diff
@@ -13,29 +13,40 @@
 # limitations under the License.
 
 import itertools
+import jinja2
 import keyword
 import re
-import gapic.samplegen.utils
+
+from gapic.samplegen import utils
 
 from collections import (defaultdict, namedtuple)
-from typing import (List, Mapping, Set)
+from textwrap import dedent
+from typing import (Dict, List, Mapping, Set, Tuple)
 
 # Outstanding issues:
-# * Neither license nor copyright are defined
 # * In real sample configs, many variables are
 #   defined with an _implicit_ $resp variable.
 
 MIN_SCHEMA_VERSION = (1, 2, 0)
 
 VALID_CONFIG_TYPE = 'com.google.api.codegen.SampleConfigProto'
 
-# There are a couple of other names that should be reserved
-# for sample variable assignments, e.g. 'client', but there are diminishing returns:
-# the sample config author isn't a pathological adversary, they're a normal person
-# who may make mistakes occasionally.
-RESERVED_WORDS = frozenset(itertools.chain(keyword.kwlist, dir(__builtins__)))
+# TODO: read in copyright and license from files.
+FILE_HEADER: Dict[str, str] = {}
```

Comment:
Could you have some placeholder values until we implement the TODO? Two reasons:

1. They will increase the odds we remember to implement the TODO before GA (especiallyif you make the placeholders really LOUD, and maybe with a special key phrase like `TODO` or `FIXME`
2. Until we implement the TODO, we can still see how the header will show up in the generated artifacts.

Reply:
Added them back with loud, dummy contents.